torch/csrc/jit/ivalue.h

```diff
@@ -190,6 +208,27 @@ struct IValue {
     JIT_ASSERT(isInt());
     return as_int;
   }
+  std::string formatInt() const {
+    JIT_ASSERT(isInt());
+    std::ostringstream ss;
+    ss << as_int;
+    return ss.str();
+  }
+
+  // String
+  IValue(std::string s)
+  : tag(Tag::String), retainable(false) {
+    as_string = s;
+  }
+  bool isString() const { return Tag::String == tag; }
+  std::string toString() const {
+    JIT_ASSERT(isString());
+    return as_string;
+  }
+  std::string formatString() const {
+    JIT_ASSERT(isString());
+    return as_string;
```

Comment:
can we escape it here?

Reply:
What is the purpose of parsing the whitespace if we always convert it back? I think we still want it for print output.